server/channels/api4/access_control.go

```diff
@@ -56,6 +56,13 @@ func createAccessControlPolicy(c *Context, w http.ResponseWriter, r *http.Reques
 		hasManageSystemPermission := c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionManageSystem)
 
 		if !hasManageSystemPermission {
+			// FEATURE_FLAG_REMOVAL: ChannelAdminManageABACRules - Remove this check when feature is GA
+			if !c.App.Config().FeatureFlags.ChannelAdminManageABACRules {
+				c.SetPermissionError(model.PermissionManageSystem)
+				return
```

Comment:
We should return `501` here, no?

Reply:
Subsequent PR is okay with a ticket to follow up. This is technically incorrect but should have big change on UX.

Reply:
ok, I will for sure change this in one of the next Prs that are coming that are part of the feature.